packages/core/src/dialects/postgres/query-interface.d.ts

```diff
@@ -1,9 +1,20 @@
-import type { Sequelize } from '../../sequelize.js';
+import type { QueryRawOptions, Sequelize } from '../../sequelize.js';
 import { AbstractQueryInterface } from '../abstract/query-interface.js';
 import type { PostgresQueryGenerator } from './query-generator.js';
 
+export type SchemaOption = {
```

Comment:
This SchemaOption type does not seem to be used

Reply:
Removed this.